tools/tdbg/adminClusterSearchAttributesCommands.go

```diff
@@ -0,0 +1,260 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package tdbg
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/olekukonko/tablewriter"
+	"github.com/urfave/cli/v2"
+	enumspb "go.temporal.io/api/enums/v1"
+
+	"go.temporal.io/server/api/adminservice/v1"
+	clispb "go.temporal.io/server/api/cli/v1"
+)
+
+const (
+	addSearchAttributesTimeout = 30 * time.Second
+)
+
+// AdminAddSearchAttributes to add search attributes
+func AdminAddSearchAttributes(c *cli.Context) error {
```

Comment:
search attribute related commands should also be part of tctl and access via Operator API. (The API is already available)

Reply:
Created them under `tctl cluster` using Operator API https://github.com/temporalio/tctl/pull/218
Removed from tbdg